packages/node_modules/@ciscospark/react-component-utils/src/constants.js

```diff
@@ -68,3 +68,12 @@ export const SPACE_TYPE_ONE_ON_ONE = 'direct';
 export const SPACE_TYPE_GROUP = 'group';
 
 export const TAG_MUTED = 'MUTED';
+
+export const attachmentType = 'AdaptiveCard';
+export const cardReference = 'cardReference';
+export const cards = 'cards';
+
```

Comment:
Please stick with the all uppercase naming convention for constants. They should also have a bit more descriptive name so that developers can understand what they are used for.

Reply:
Changed constants with proper description names and to upper case